docs/content/_navigation.json

```diff
@@ -696,6 +696,10 @@
             "title": "Moving to Software-defined assets",
             "path": "/guides/dagster/enriching-with-software-defined-assets"
           },
+          {
+            "title": "Migrating to Pythonic resouces and config",
```

Comment:
based on the diff here it looks like this is under the general guides, but in the vercel preview it's showing as part of the experimental guides. Just want to confirm where this page is living, since to me, I think it makes sense to have it in the general guides, as it's no longer an experimental feature